distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -84,14 +84,21 @@ org.codice.ddf.system.registry-id=
 # Uncomment or comment the properties based on the desired configuration:
 #
 
-# Solr Server Provider
+# Solr Server Authentication
+solr.useBasicAuth=true
+solr.username=ddf
+solr.credentials=changeit
```

Comment:
❓ is `solr.credentials` meaning "password"? Any reason not to use `password` ?

Reply:
Its even referred to as "password" in other comments 😄 
https://github.com/codice/ddf/pull/3746/files#r217875209

Reply:
The static analysis checkers often freak out when they see "password" anywhere. I wanted to avoid that. I choose "credentials" because Solr uses that word instead of "password" in the `security.json` file.

Reply:
Imo changing the name to avoid static analysis is redundant because you're still putting a password in here -- which i presume is what static analysis is advocating for. I don't think they necessarily have anything against the name password.

Reply:
Some static analysis tools are...not so smart. I know for example that Fortify will flag fields on classes with the string `password` in their names, even if those fields merely refer to system properties named password. I don't recall seeing any of those tools that looked at properties files, but if the property file has a key named `foo.password`, that implies that some Java code somewhere in the application will have a lookup for `foo.password` which might be enough to confuse tools like that.

Whether it's important enough to try to obfuscate rather than marking as not-a-finding is a coin toss.

Reply:
If there is passion about it, I will change it. 
If there is no passion, I'll leave it as-is.
Give me a thumbs up on the comment to indicate you want "credentials" changed to "password".

Reply:
Since there other entries in system.properties named password. I vote for password and be consistent.